src/lib/modules.ts

```diff
@@ -454,6 +454,399 @@ export const wysiwygTextModuleSchema = z.object({
 })
 export type WysiwygTextModuleData = z.infer<typeof wysiwygTextModuleSchema>
 
+export const triMobileAnimatedModule = {
+  name: 'TriMobileAnimatedModule',
+  label: 'Tri Mobile Animated',
+  widget: 'object',
+  fields: [
+    {
+      ...mediaComponent,
+      label: 'Media Background',
+      name: 'mediaBackground',
+    },
+    {
+      ...mediaComponent,
+      label: 'Media 1',
+      name: 'media1',
+    },
+    {
+      ...mediaComponent,
+      label: 'Media 2',
+      name: 'media2',
+    },
+    {
+      ...mediaComponent,
+      label: 'Media 3',
+      name: 'media3',
+    },
+  ],
+} as const //  satisfies Immutable<CmsField>
+export const triMobileAnimatedModuleSchema = z.object({
+  type: z.literal(triMobileAnimatedModule.name),
+  mediaBackground: mediaComponentSchema,
+  media1: mediaComponentSchema,
+  media2: mediaComponentSchema,
+  media3: mediaComponentSchema,
+})
+export type TriMobileAnimatedModuleData = z.infer<typeof triMobileAnimatedModuleSchema>
+
+export const deviseColumnsModule = {
+  name: 'DeviseColumnsModule',
+  label: 'Devise Columns',
+  widget: 'object',
+  fields: [
+    {
+      label: 'Items',
+      name: 'items',
+      widget: 'list',
+      min: 1,
+      max: 2,
+      fields: [
+        {
+          label: 'Has Solid Background',
+          name: 'hasSolidBackground',
+          widget: 'boolean',
+          default: false,
+          required: true,
+        },
+        {
+          label: 'Background Color',
+          name: 'backgroundColor',
+          widget: 'string',
+          default: '#000000',
+        },
+        {
+          ...mediaComponent,
+          label: 'Media Background',
+          name: 'mediaBackground',
+          required: false,
+        },
+        {
+          label: 'Has Overlay',
+          name: 'hasOverlay',
+          widget: 'boolean',
+          default: false,
+          required: true,
+        },
+        {
+          label: 'Overlay Type',
+          name: 'overlayType',
+          widget: 'select',
+          options: [
+            { label: 'Mobile Wrapper', value: 'mobile-wrapper' },
+            { label: 'Desktop Wrapper', value: 'desktop-wrapper' },
+            { label: 'Square', value: 'aspect-1-1' },
+            { label: 'Landscape', value: 'aspect-16-9' },
+            { label: 'Rectangle', value: 'aspect-4-3' },
+          ],
+          default: 'mobile-wrapper',
+        },
+        {
+          ...mediaComponent,
+          label: 'Media Overlay',
+          name: 'mediaOverlay',
+        },
+      ],
+    },
+  ],
+} as const //  satisfies Immutable<CmsField>
+export const deviseColumnsModuleSchema = z.object({
+  type: z.literal(deviseColumnsModule.name),
+  items: z.array(
+    z.object({
+      hasSolidBackground: z.boolean().optional(),
+      backgroundColor: z.string().optional(),
```

Comment:
```suggestion
      backgroundColor: z.string(),
```
Values with default don't need optional